test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -48,4 +48,17 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $output = 1 | ConvertTo-Json
         $output | Should -BeExactly '1'
     }
+
+    It "The result string should be escaped." {
+        if ($isWindows) {
+            $newline = "`r`n"
+        } else {
+            $newline = "`n"
+        }
+        $expected1 = "{$newline  ""abc"": ""'def'""$newline}"
+        $expected2 = "{$newline  ""abc"": ""\u0027def\u0027""$newline}"
+        @{ 'abc' = "'def'" } | ConvertTo-Json | Should -BeExactly $expected1
```

Comment:
Turn this into `-TestCases`?  Benefit is if first assert fails, the whole thing doesn't fail.